plugins/modules/vmware_guest_controller.py

```diff
@@ -108,6 +108,13 @@
          choices:
            - present
            - absent
+       bus_sharing:
+         description:
+         - Bus sharing type for SCSI controller.
+         required: false
+         type: str
+         choices: ['noSharing', 'physicalSharing', 'virtualSharing' ]
+         version_added: '1.8.0'
```

Comment:
```suggestion
         version_added: '1.11.0'
```

We're on 1.10 already, but maybe we'll get it into 1.11 :-)